RevenueCatUI/Templates/V2/Components/Stack/StackComponentView.swift

```diff
@@ -90,13 +90,10 @@ struct StackComponentView: View {
         }
         .padding(style.padding)
         .padding(additionalPadding)
-        .backgroundStyle(style.backgroundStyle)
         .shape(border: style.border,
-               shape: style.shape)
-        .applyIfLet(style.shadow) { view, shadow in
-            // Without compositingGroup(), the shadow is applied to the stack's children as well.
-            view.compositingGroup().shadow(shadow: shadow)
-        }
+               shape: style.shape,
+               shadow: style.shadow,
+               background: style.backgroundStyle)
```

Comment:
The `border` already depended on the `shape`, and now the `shadow` does as well, so we include it in the `shape` modifier.

Since the `shape` modifier already does shape clipping of the background, and requires the `backgroundStyle` modifier to be applied right before, it makes sense to move it inside the `shape` modifier as well so we can more easily track the interactions between all the pieces.